Package.swift

```diff
@@ -0,0 +1,39 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "CocoaLumberjack",
+    platforms: [
+        .iOS(.v8),
+        .macOS(.v10_10),
+        .watchOS(.v3),
+        .tvOS(.v9)
+    ],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "CocoaLumberjack",
+            targets: ["CocoaLumberjack"]),
+        .library(
+            name: "CocoaLumberjackSwift",
+            targets: ["CocoaLumberjackSwift"]),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(name: "CocoaLumberjack",
+                exclude: ["Supporting Files"]),
+        .target(name: "CocoaLumberjackSwiftSupport",
+                dependencies: ["CocoaLumberjack"]),
+        .target(name: "CocoaLumberjackSwift",
+                dependencies: ["CocoaLumberjack", "CocoaLumberjackSwiftSupport"],
+                exclude: ["Supporting Files"]),
+        .testTarget(name: "Tests",
+                    dependencies: ["CocoaLumberjack"],
+                    path: "Tests",
```

Comment:
Do we need `path`, `exclude`, and `sources`? If we name the folders correctly, SPM should catch this automatically.

Reply:
@ffried 
For that we need to `move Tests -> Sources/CocoaLumberjackTests`, right?

Reply:
Nope. Tests are always searched inside the "Tests" folder. So you need to have a folder "CocoaLumberjackTests" inside "Tests" and it'll automatically pick those up.

Reply:
@ffried 
I don't remember if it is required or not to include `.testTarget()` in `.library()` in SPM.

Reply:
@lolgear You mean if you need to define a product for it? You don't. 
Targets are internal and don't leave the package. Products are what leave the package. But tests should never leave the package. They should only be internal to the package.